app/javascript/chat/channels.jsx

```diff
@@ -80,19 +80,23 @@ export const Channels = ({
   }
   let configFooter = '';
   if (expanded) {
-    // TODO: The <div /> below should be converted into a real menu or <nav />
     configFooter = (
-      <div className="chatchannels__config">
-        <img alt="configration" src={ConfigImage} style={{ height: '18px' }} />
-        <div className="chatchannels__configmenu" role="menu">
-          <a href="/settings" role="menuitem">
-            Settings
-          </a>
-          <a href="/report-abuse" role="menuitem">
-            Report Abuse
-          </a>
-        </div>
-      </div>
+      <nav className="chatchannels__config" aria-label="configuration menu">
+        <img
+          alt="configuration"
+          src={ConfigImage}
+          style={{ height: '18px' }}
+          aria-hidden="true"
+        />
+        <ul className="chatchannels__configmenu">
+          <li>
+            <a href="/settings">Settings</a>
+          </li>
+          <li>
+            <a href="/report-abuse">Report Abuse</a>
+          </li>
+        </ul>
+      </nav>
```

Comment:
Although this change correctly changes the group to a `nav`, it's still not accessible to assistive technology users. The nav items are rendered as hidden, and the menu can't be activated without using a mouse. The result is now instead of this whole block of content being hidden from screen reader users, now we tell them there is a `nav`, but it's empty 😞

See below: 

![VoiceOver rotor showing a list including empty nav](https://user-images.githubusercontent.com/20773163/107018210-0c100b00-6798-11eb-9e1b-c88f7b52030b.gif)